packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -214,9 +216,43 @@ class TextMapPropagator {
     return this._config.tracePropagationStyle[mode].includes(name)
   }
 
+  _resolveTraceContextConflicts (w3cSpanContext, firstSpanContext, carrier) {
```

Comment:
This function would be a lot easier to read if split up with smaller functions with representative names that would also replace comments. For example `hasTraceParent` etc.